lib/ProductOpener/Ingredients.pm

```diff
@@ -154,6 +154,7 @@ sub compute_carbon_footprint_from_ingredients($) {
 		delete $product_ref->{nutriments}{"carbon-footprint-from-meat-or-fish_product"};
 	}
 
```

Comment:
Could you add a comment here too?

Reply:
Same thing, we delete old values before computing new ones.